tests/UnitTests/test_1password_api.py

```diff
@@ -0,0 +1,73 @@
+from datetime import datetime, UTC, timedelta
+import json
+from os.path import abspath, dirname
+from pydantic import ValidationError
+import responses
+import unittest
+
+from src.apis.general.Api import ReqMethod
+from src.apis.onepassword.OnePassword import OnePassword
+
+curr_path = abspath(dirname(__file__))
+
+
+class TestOnePasswordApi(unittest.TestCase):
```

Comment:
We need to decide on one naming convention :) `onepassword` or `1password` across class names and properties, tests and file names

Reply:
I'll change the file name 🙏🏼